src/lib.rs

```diff
@@ -89,6 +89,9 @@ pub extern "C" fn ckb_exec_cell(
     argv: *const *const u8,
 ) -> c_int {
     assert_vm_version();
+    if argc == 0 {
```

Comment:
In our design, `argc` in exec can be zero.  It's different than design in unix systems.

Reply:
Here, we can use linux syscall directly. 
I think it's not necessary to implement exec in simulator currently.

Reply:
Currently implementation(use `libc::execvp`) tested ok in ckb-std test contract, may revisit this PR in future?